apps/ima/index.js

```diff
@@ -33,13 +38,68 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
-      backLink: 'current-progress'
+      backLink: 'current-progress',
+      next: '/evidence-upload'
+    },
+    '/evidence-upload': {
+      behaviours: [SaveImage('image'), RemoveImage, LimitDocument],
+      fields: ['image'],
+      continueOnEdit: true,
+      next: '/confirm'
     },
     '/confirm': {
       behaviours: [Summary, SaveFormSession],
       sections: require('./sections/summary-data-sections'),
       locals: { showSaveAndExit: true },
+      forks: [ // Update to late submission step
+        {
+          target: '/declaration-person-named',
+          condition: {
+            field: 'who-are-you',
+            value: 'person-named'
+          }
+        },
+        {
+          target: '/declaration-immigration-adviser',
+          condition: {
+            field: 'who-are-you',
+            value: 'has-legal-representative'
+          }
+        },
+        {
+          target: '/declaration-someone-else',
+          condition: {
+            field: 'who-are-you',
+            value: 'someone-else'
+          }
+        }
+      ]
+    },
+    '/declaration-person-named': {
+      behaviours: [Summary, Submit],
+      sections: require('./sections/summary-data-sections'),
+      fields: [
+        'declaration-person-named'
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/confirmation'
+    },
+    '/declaration-immigration-adviser': {
+      behaviours: [Summary, Submit],
+      sections: require('./sections/summary-data-sections'),
+      fields: [
+        'declaration-immigration-adviser'
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/confirmation'
+    },
+    '/declaration-someone-else': {
```

Comment:
Path names do not match those in the latest Figma designs.

Reply:
Path names changed as per Figma designs.